autoload/fzf/bookmarks.vim

```diff
@@ -0,0 +1,126 @@
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+" color functions
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+function! s:get_color(attr, ...)
+    let gui = has('termguicolors') && &termguicolors
+    let fam = gui ? 'gui' : 'cterm'
+    let pat = gui ? '^#[a-f0-9]\+' : '^[0-9]\+$'
+    for group in a:000
+        let code = synIDattr(synIDtrans(hlID(group)), a:attr, fam)
+        if code =~? pat
+            return code
+        endif
+    endfor
+    return ''
+endfunction
+
+if &t_Co == 256
+    let s:ansi = {'black': 234, 'red': 196, 'green': 46, 'yellow': 178, 'blue': 63, 'magenta': 201, 'cyan': 51}
+else
+    let s:ansi = {'black': 30, 'red': 31, 'green': 32, 'yellow': 178, 'blue': 34, 'magenta': 35, 'cyan': 36}
+endif
+
+function! s:csi(color, fg)
+    let prefix = a:fg ? '38;' : '48;'
+    if a:color[0] == '#'
+        return prefix.'2;'.join(map([a:color[1:2], a:color[3:4], a:color[5:6]], 'str2nr(v:val, 16)'), ';')
+    endif
+    return prefix.'5;'.a:color
+endfunction
+
+function! s:ansi(str, group, default, ...)
+    let fg = s:get_color('fg', a:group)
+    let bg = s:get_color('bg', a:group)
+    let color = s:csi(empty(fg) ? s:ansi[a:default] : fg, 1) .
+                \ (empty(bg) ? '' : s:csi(bg, 0))
+    return printf("\x1b[%s%sm%s\x1b[m", color, a:0 ? ';1' : '', a:str)
+endfunction
+
+for s:color_name in keys(s:ansi)
+    execute "function! s:".s:color_name."(str, ...)\n"
+                \ "  return s:ansi(a:str, get(a:, 1, ''), '".s:color_name."')\n"
+                \ "endfunction"
+endfor
+
+
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+" format functions
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+function! s:pad(t, n)
+    if len(a:t) > a:n
+        return a:t[:(a:n-1)]."…"
+    else
+        let spaces = a:n - len(a:t)
+        let spaces = printf("%".spaces."s", "")
+        return a:t.spaces
+    endif
+endfunction
+
+function! s:is_annotation(t)
+    if a:t =~ "^Annotation: " | return s:green("§ ".a:t[12:]) | endif
```

Comment:
I'm removing `"Annotation: "` to only show the annotation string, and marking annotations with the symbol `"§"`. If you prefer another symbol, you can replace it (an unicode icon could not be typed, so there would be no easy way to filter annotations). If you change the length of the symbol (by using `"!!! "`, for example) you should also change below to keep the alignment.